portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,12 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+const facts = [
+  'i love cats', 
+  'i love programming', 
+  'i love sundays', 
+  'i love cakes'
```

Comment:
please add a training comma to the last element of a list:
https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma